verifiable-db/src/cells_tree/mod.rs

```diff
@@ -33,7 +33,7 @@ pub use public_inputs::PublicInputs;
 /// A cell represents a column || value tuple. it can be given in the cells tree or as the
 /// secondary index value in the row tree.
 #[derive(Clone, Debug, Serialize, Deserialize, Constructor)]
-pub(crate) struct Cell {
+pub struct Cell {
```

Comment:
Why making these public? No issue in making them public, just curious to learn why clippy was complaining :)

Reply:
So clippy tends to complain if the visibility is at `crate` level if you don't then use the object from within the same package (excluding tests). However if you make it fully public, so anyone using `verifiable_db` as a library has access it doesn't complain because it thinks "Oh somebody importing this library might use it so i won't warn them that they don't use it here". The methods `digest`, `split_digest` and `split_and_accumulate_digest` of `Cells` were never used, but I thought they might get used in the future so rather than delete them the cleaner option seemed to be making the struct and those methods fully public.

Reply:
Makes sense to keep them, they might be actually used in the future